tests/snapshots/tests.integration.test_integration_snapshots.test_tracer_trace_across_fork.json

```diff
@@ -30,7 +30,7 @@
          "runtime-id": "409d5f5531a54659b762419af6a3abc5"
        },
        "metrics": {
-         "_dd.top_level": 0,
+         "_dd.top_level": 1,
```

Comment:
this is the trade off we are making where this will get marked as top level since the it is the local root in the fork.

the service naming being null is... potentially a problem. cc @Kyle-Verhoog

Reply:
service being null here is because we never set one in the test case, we probably should though to ensure that it does (or doesn't get propagated).

I don't think it does today which isn't great...